lua/refactoring/code_generation/go.lua

```diff
@@ -41,6 +41,13 @@ local function constant(opts)
 end
 
 local go = {
+    print_var = function(prefix, var)
+        return string.format(
+            'fmt.Println(fmt.Sprintf("%s %%v", %s))',
```

Comment:
%v or %+v

Reply:
I believe `%v` is the correct thing for golang and `%%v` keeps the `%` when formatting strings in lua.